pkg/registry/registry.go

```diff
@@ -196,5 +197,10 @@ func IsOktetoRegistry(tag string) bool {
 }
 
 func replaceRegistry(input, registryType, namespace string) string {
-	return strings.Replace(input, registryType, fmt.Sprintf("%s/%s", okteto.Context().Registry, namespace), 1)
+	// Check if the registryType is the start of the sentence or has a whitespace before it
+	var re = regexp.MustCompile(fmt.Sprintf(`(^|\s)(%s)`, registryType))
```

Comment:
The regex compilation should probably be moved to package level. This will both cause the panic before the function is called, making detecting an issue easier, and save some performance by only compiling the regex once.